Sources/XiEditor/Annotation.swift

```diff
@@ -54,7 +54,7 @@ typealias AnnotationSpan = (startIx: Int, endIx: Int, annotation: Annotation)
 struct AnnotationStore {
     var annotations: [AnnotationType: [Annotation]]
 
-    init(from json: [[String: AnyObject]]) {
```

Comment:
This is a bit nitpicky.  We often cast to `[String: AnyObject]` but then pull out primitive types from the cast object. `AnyObject` is a the most generic class type whereas `Any` can be anything.  

Happy to reverse these changes if folks feel strongly about keeping it as `AnyObject`.